### ISSUE TITLE
Feature request: Add accessibility for blind players.

### ISSUE THREAD

**Issue by germanico5** [yairm210/Unciv#12925]
### Before creating

- [x] This is NOT a gameplay feature from Civ VI, BNW, or outside - see Roadmap
- [x] This is NOT a gameplay feature from Vanilla Civ V or from G&K - If so, it should be a comment in https://github.com/yairm210/Unciv/issues/4697

### Problem Description

The game currently cannot be played by blind people. It would need to have compatibility with screen readers, whether system-based like TalkBack or directly implemented into the game. The movement system would also likely need to be adapted, as the hexagonal board is difficult for blind users to navigate.

### Desired Solution

Users would need to be able to navigate menus, inspect the board, and play normally with the screen reader being able to read the game's content. I came here for the mobile version, but if it's on PC, it might be easier to implement. I think the most challenging part could be the aforementioned board movement, but I am willing to provide ideas and test everything necessary.

### Alternative Approaches

-

**Comment by yairm210** [comment]
I have absolutely no idea how to go about this. This is an interesting ask, but totally out of scope.
Do you have an example of any other hexagonal map 4x game that does this?

**Comment by germanico5** [comment]
Honestly, there isn’t much out there. Not because it’s impossible, but because it’s not common for developers to fully commit to accessibility projects, which I find reasonable. The only games I’ve seen adapt a hexagonal board are board games, but I completely dismiss what they did because it would be too slow for a 4X game. What I can think of regarding movement is simply making it grid-based, where moving up doesn’t take you to the hexagon directly above but to one of the diagonally adjacent ones. This would be the "easiest" approach before innovating with movement. It would be amazing if a game like this could be made accessible. I’m not a programmer, but I have experience as a blind player to help guide you.

**Comment by yairm210** [comment]
What it sounds like is a request to make this 100% visual game, into a non-visual game, in a way where I can't even copy from anyone else because no one has done this :|
And change the hexagonal map into a grid map
Waaay out of scope for me, sorry
